lib/connect.ts

```diff
@@ -1319,8 +1319,8 @@ regular).
     function transformMFInstrumentsResponse(data, headers) {
         // Parse CSV responses
         if (headers["content-type"] === "text/csv") {
-            var parsedData = csvParse.parse(data, {"header": true}).data;
-            for (var item of parsedData) {
+            const parsedData = csvParse.parse(data, {"header": true}).data;
+            for (const item of parsedData as any) {
```

Comment:
If we use @bhavya3024 any all the places no need to use `TypeScript`. if we use appropriate Types So we will track all our objects for particular objects.